docs/docsite/rst/playbooks_lookups.rst

```diff
@@ -20,8 +20,7 @@ Lookups and loops
 
 *lookup plugins* are a way to query external data sources, such as shell commands or even key value stores.
 
-Before Ansible 2.5, lookups were mostly used indirectly in ``with_<lookup`` constructes for looping, begining in 2.5
-we use them more explicitly as part of Jinja2 expressions fed into the ``loop`` keyword.
+Before Ansible 2.5, lookups were mostly used indirectly in ``with_<lookup`` constructs for looping. Starting with Ansible version 2.5, lookups are used more explicitly as part of Jinja2 expressions fed into the ``loop`` keyword.
```

Comment:
The ```with_<lookup``` looks wrong.  Might want to ask whoever wrote this what it should be.  I think ```<lookup``` is meant to be placeholder meaning substitute your variable here.  So perhaps ``` ``with_<lookup>`` ``` (although that may conflict with rst syntax.  If it does then maybe ``` ``with_$LOOKUP`` ```?